src/lightning_app/frontend/panel/panel_frontend.py

```diff
@@ -50,7 +50,7 @@ class PanelFrontend(Frontend):
     .. code-block:: python
 
         import lightning as L
-        from lightning.app.frontend.panel import PanelFrontend
+        from lightning_app.frontend.panel import PanelFrontend
```

Comment:
Why are these being rewritten? In all documentation and examples, we need to show the lightning import. Here with the frontend, the problem is that frontend is not a top-level importable module. Either frontend has to be added in `__init__.py` or change the import to import `lightning.app.frontend` as frontend or similar. 


See also #15096

Reply:
@awaelchli this way lightning_app could theoretically be installed as a standalone package. This is required for the import-converter to work properly and to be able to test the packages independently. In the end the import-converter makes sure that this will be `lightning.app`

Reply:
Does the import converter somehow modify the docstrings before docs are generated?
What about the examples that people see in our GitHub examples/ directory?

Reply:
Yes, but if you install `lightning-app`, not `lightning` it will crash, in such case we deny users to install `lightning-app`

Reply:
and in the mono-package it will be back as `lightning.app`

Reply:
see #15105

Reply:
I thought that you always get the lightning meta package. If you install pytorch-lightning you get lightning.pytorch but not lightning.app. is this not what we are going for?

Reply:
not really; these are the cases:
- install `pytorch-lightning` you get `pytorch_lightning` only (now from master you get also `lightning_lite`)
- install `lightning-app` you get `lightning_app` only
- install `lightning` you get all there:
  * `pytorch_lightning` and `lightning.pytorch`
  * `lightning_app` and `lightning.app`